Monika After Story/game/script-topics.rpy

```diff
@@ -3586,8 +3632,19 @@ label monika_girlfriend:
     $ mas_loseAffection(reason=2)
     m 1euc "I thought we established already that I was your girlfriend?"
     m 1tkc "There's no way you already have one in your reality, right?"
-    m 4tfx "If you do, then you've got to break up with her right now!"
-    m 4hksdlb "Tell her you met someone perfect for you, someone who will never betray you!"
+    if not persistent._mas_pm_likes_yanderes:
+        m 4tfx "If you do, then you've got to break up with her right now!"
+        m 4hksdlb "Tell her you met someone perfect for you, someone who will never betray you!"
+    elif (persistent._mas_pm_likes_yanderes == -1):
+        m 1tkc "If you do, please break up with her"
+        m 1tkc "I love you, [player]"
+        m 1tkc "Please love me back"
```

Comment:
Missing a period.

I get that this is the path for if you hate yanderes, but this just sounds desperate to me. 

You also have the same expression 3 times in a row. I would mix it up a bit.

Reply:
I know this sound desperate, but I think it should. If You assume, that the previous script is what Monika wanted to say, but because she knows the player hates yanderes, she wants to make sure she doesn't sound like a yandere, but this results with her not knowing, what to say, so she sound desperate. 
Also, I think this is good point to address the thing, that this mod makes Monika yandere - I just amplified the parts of script, in which she already gave off yandere vibes, but there if the player click, that he hates yanderes, it almost completely "deyanderifies" Monia...